test/rocarray/base.jl

```diff
@@ -27,6 +27,31 @@ end
     @test collect(c) == 3:4
 end
 
+@testset "resize!" begin
+    a_h = Array(range(1, 10))
+    a_d = a_h |> roc
+    # Resize up
+    resize!(a_h, 15)
+    resize!(a_d, 15)
+    a_h[10:15] .= 15
+    a_d[10:15] .= 15
```

Comment:
I think you meant to check that `a_h[1:10] .== Array(a_d[1:10])`? The appended bytes are uninitialized, so we can't predict their values.

Reply:
I wanted to write them with the same value to check if we can write to it, and to initialize those new bytes to have predictable values.

Reply:
Right, I misread this, sorry!